Snowplow iOSTests/Legacy Tests/LegacyTestSubject.m

```diff
@@ -44,18 +44,17 @@ - (void)tearDown {
 
 - (void)testSubjectInit {
     SPSubject * subject = [[SPSubject alloc] init];
-    XCTAssertNotNil([subject getPlatformDict]);
```

Comment:
I hope this is not a breaking change that we return nil from getPlatformDict in case platformContext setting is false in tracker configuration – the default initializer sets it to false.